security/bouncycastle-fips/src/test/java/io/quarkus/ts/security/bouncycastle/fips/BouncyCastleFipsJsseIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.security.bouncycastle.fips;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Dependency;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class BouncyCastleFipsJsseIT {
+
+    private static final String PASSWORD = "password";
+
+    @QuarkusApplication(dependencies = {
+            @Dependency(groupId = "org.bouncycastle", artifactId = "bctls-fips", version = "1.0.12.2")
```

Comment:
why is explicit dependency definition needed? Assuming that the dependency contains BCJSSE provider. 

Is explicit version needed?

Reply:
yes, unfortunately, Quarkus test framework doesn't resolve thrid party dependencies dynamically. 

Ref: https://github.com/quarkus-qe/quarkus-test-framework/issues/453